sandbox/drone-imagery-ingestion/full-run-configs/base.yml

```diff
@@ -129,7 +142,16 @@ buildDem: # (Metashape: buildDem, (optionally) classifyGroundPoints, exportRaste
 
 buildOrthomosaic: # (Metashape: buildOrthomosaic, exportRaster)
     enabled: True
-    surface: ["DSM-mesh"] # Options: "DTM-ptcloud", "DSM-ptcloud", "DSM-mesh", and/or "Mesh". The surface to build the orthomosaic onto. DTM and DSM refer to elevation models built by Metashape and must be configured to be computed via buildDem, above. Mesh refers to using the mesh model directly rather than first computing a DEM.
+    # TODO This is a very annoying hack because the R yaml parser simplifies one-length lists into a
+    # single element. To get around this, we duplicate the element, running the same operation
+    # twice. Fortunately, this operation is relatively quick in the scheme of things.
+    # See discussions below.
+    # https://stackoverflow.com/questions/23475743/yaml-in-r-passing-in-a-one-element-list
+    # https://github.com/vubiostat/r-yaml/issues/69
+    # Options: "DTM-ptcloud", "DSM-ptcloud", "DSM-mesh", and/or "Mesh". The surface to build the orthomosaic onto. DTM and DSM refer to elevation models built by Metashape and must be configured to be computed via buildDem, above. Mesh refers to using the mesh model directly rather than first computing a DEM.
+    surface:
+        - DSM-mesh
+        - DSM-mesh
```

Comment:
I see this is duplicated as part of a necessary hack. However maybe we could also generate a `DSM-ptcloud` (I think these actually look better, something I realized after running the other projects with only `DSM-mesh`)

Reply:
As a side comment, is it a problem that this list gets simplified to a single element? Does that break automate-metashape?

Reply:
That sounds like a good alternative, I was just trying to keep the outputs consistent with the past. Yes, the issue is with automate-metashape. If surface(s) are represented as a single string, it tries to iterate over the characters in that string and looks for the surface `D`.